tests/plugins/balloontoolbar/plugin.js

```diff
@@ -0,0 +1,26 @@
+/* bender-tags: balloontoolbar, 1346, context */
```

Comment:
I think `context` it to detailed, I would go with `editor, balloontoolbar, 1346`.